_schema.graphql

```diff
@@ -4084,6 +4092,11 @@ union Item = ArtistItem | ArtworkItem | FeaturedLinkItem | GeneItem
 # The `JSON` scalar type represents JSON values as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf).
 scalar JSON
 
+type LatLng {
+  lat: Float
+  lng: Float
+}
```

Comment:
This already exists as "coordinates" (who made a lowercase type?!)

https://github.com/artsy/metaphysics/blob/5659ff0807cca37705520b47124d51f62c8de873/_schema.graphql#L2285-L2288

Reply:
Good catch 😄. Looks like that was added 3 years ago(!) and is only used in one place.

Question: does changing the name of this type (even if only for case) qualify as a deprecation, in graphql terms?

Reply:
Hmm, I think it’s never safe, actually. There’s many places in our client codebases where we have fragments (which are always on types), so if any of our clients has a fragment on `coordinates` then I’m pretty sure this will break (unverified, though!)

Reply:
10/10 @alloy  - great point!

I did a [quick look](https://github.com/search?q=org%3Aartsy+%22on+coordinate%22&type=Code) and we don't have it anywhere, so it is good to transform, but realistically we can't do it very often

Reply:
👍